avocado/core/runners/requirement_package.py

```diff
@@ -0,0 +1,133 @@
+import time
+from multiprocessing import Process, SimpleQueue
+
+from ...utils.software_manager.main import MESSAGES
+from ...utils.software_manager.manager import SoftwareManager
```

Comment:
As I mentioned on the previous version, we could remove the selftests along with those relative references. But I understand how important is this feature and I will not block the PR because of this.